src/styles/blocks/card.scss

```diff
@@ -0,0 +1,124 @@
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+
+  padding: 50px 0 0;
+
+  margin: 0 auto;
+
+  max-width: 944px;
+}
+
+.card {
+  width: 200px;
+  height: 406px;
```

Comment:
Don't hardcode exact height size. Add necessary paddings and margins if u need and let content dictate the container size.